contributors/guide/pull-requests.md

```diff
@@ -25,11 +25,22 @@ This doc explains the process and best practices for submitting a pull request t
   - [3. Open a Different Pull Request for Fixes and Generic Features](#3-open-a-different-pull-request-for-fixes-and-generic-features)
   - [4. Comments Matter](#4-comments-matter)
   - [5. Test](#5-test)
-  - [6. Squashing and Commit Titles](#6-squashing-and-commit-titles)
-  - [7. KISS, YAGNI, MVP, etc.](#7-kiss-yagni-mvp-etc)
-  - [8. It's OK to Push Back](#8-its-ok-to-push-back)
-  - [9. Common Sense and Courtesy](#9-common-sense-and-courtesy)
-  - [10. Trivial Edits](#10-trivial-edits)
+  - [6. Squashing](#6-squashing)
+  - [7. Commit Message Guidelines](#7-commit-message-guidelines)
+    - [Try to keep the subject line to 50 characters or less; do not exceed 72 characters](#try-to-keep-the-subject-line-to-50-characters-or-less-do-not-exceed-72-characters)
+    - [The first word in the commit message subject should be capitalized unless it starts with a lowercase symbol or other identifier](#the-first-word-in-the-commit-message-subject-should-be-capitalized-unless-it-starts-with-a-lowercase-symbol-or-other-identifier)
+    - [Do not end the commit message subject with a period](#do-not-end-the-commit-message-subject-with-a-period)
+    - [Use imperative mood in your commit message subject](#use-imperative-mood-in-your-commit-message-subject)
+    - [Add a single blank line before the commit message body](#add-a-single-blank-line-before-the-commit-message-body)
+    - [Wrap the commit message body at 72 characters](#wrap-the-commit-message-body-at-72-characters)
+    - [Do not use GitHub keywords or (@)mentions within your commit message](#do-not-use-github-keywords-or-mentions-within-your-commit-message)
+      - [GitHub Keywords](#github-keywords)
+      - [(@)Mentions](#mentions)
+    - [Use the commit message body to explain the _what_ and _why_ of the commit](#use-the-commit-message-body-to-explain-the-what-and-why-of-the-commit)
```

Comment:
These were added by auto-toc - I'm not quite sure how I feel about them being in here. What are other folks opinions?

Reply:
Strange...Why would the bullet list interpret as headings?

Reply:
Updated headings so auto-toc doesn't include them 👍